src/git/extension.ts

```diff
@@ -167,6 +178,36 @@ export class Extension {
         // or if we no longer have focus on any file
         if (before === after || after === NO_FILE_OR_PLACE) {
             this.view.set(lineAware?.commit);
+
+            if (lineAware?.commit) {
+                let decorationText = toTextView(lineAware.commit);
+                let margin = getProperty("inlineBlameMargin") ?? 2;
+                //let decorationColor = getProperty("inlineBlameColor") ?? "#888987";
+
+                //clear old decorations
+                textEditor.setDecorations?.(this.decorationType, []);
+                //add new decoration
+                textEditor.setDecorations?.(
+                    this.decorationType,
+                    [
+                        {
+                            renderOptions: {
+                                after: {
+                                    contentText: decorationText,
+                                    margin: `0 0 0 ${margin}rem`,
+                                    color: new ThemeColor("gitblame.inlineBlameColor"),
```

Comment:
I think this misinterprets how the `ThemeColor` works. The `id` should be [from this list](https://code.visualstudio.com/api/references/theme-color). I would personally chose `editorCodeLens.foreground`.

If we do that we also don't need a setting for it and that can be removed.